docs/docgen.md

```diff
@@ -29,6 +29,19 @@ implicitly change its status to `NotYetImplemented`.
 One day we could (and should!) generate boilerplate for undocumented methods.
 For now, it's hard to tell which section these should go in and so we skip this.
 
+## Macros
+
+Throughout the documentation (in pages, in the summary page, and
+elsewhere), various macros are available.
```

Comment:
Yes - did you want to do that? There are a fair few macros cropping up - we could have a separate page for each parameter, but flipping between them might be  horrid? What do you prefer?

(in terms of docgen complexity, having separate pages would involve writing a mechanism for separate pages, which I've been slightly trying to avoid because it's work, but it's not _hard_ work .. )

If you're concerned about space on the page, we could make the macros into popups in some way? @Lama1403 might have views on that though?

Reply:
(or I guess we could have a single page with all parameters, but some are subtly different - eg. transactions you send and transactions you receive - which could be fine, or not fine, depending on what you think!)

Reply:
Moving to slack so I can resolve this conversation ..